### PR TITLE
[stable/opa] add timeout for a webhook call in seconds

### PR DESCRIPTION
**Pull request by etharendil**
Signed-off-by: Ondrej Kuchar <ondrej.kuchar@pan-net.eu>

#### What this PR does / why we need it:
Timeout for a webhook call in seconds. Starting in kubernetes 1.14 you can set the timeout and it is encouraged to use a small timeout for webhooks. If the webhook call times out, the request the request is handled according to the webhook'sfailure policy.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by etharendil** [comment]
I'm not able to assign the PR. 
cc @tsandall @timothyhinrichs  please have a look

File: stable/opa/templates/webhookconfiguration.yaml [hunk 1]
```diff
@@ -32,6 +32,9 @@ webhooks:
         name: {{ template "opa.fullname" . }}
         namespace: {{ .Release.Namespace }}
     sideEffects: {{ .Values.admissionControllerSideEffect }}
+{{ if .Values.timeoutSeconds }
```
Comment (zanhsieh) [inline stable/opa/templates/webhookconfiguration.yaml]: Missing a '}' in the line end.

Comment (etharendil) [reply stable/opa/templates/webhookconfiguration.yaml]: fixed, thank you

**Review by zanhsieh — APPROVED** [review]

**Comment by zanhsieh** [comment]
/assign @timothyhinrichs
@timothyhinrichs 
@tsandall 
Would you mind to LGTM this please? Thank you.

**Pull request merged**